### PR TITLE
Apache supports TLSv1.3

### PR DESCRIPTION
**Pull request by JBlond**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -158,8 +158,8 @@ <h3 id="server-performance"><a href="#server-performance">Server performance</a>
             <td class="ok"><a href="http://httpd.apache.org/docs/trunk/mod/mod_ssl.html#sslalpnpreference">yes</a></td>
             <td class="ok"><a href="https://wiki.mozilla.org/Security/Server_Side_TLS#Apache">yes</a></td>
             <td class="ok"><a href="https://httpd.apache.org/docs/trunk/mod/mod_http2.html">yes</a></td>
-            <td class="alert">no</td>
-            <td class="alert">no</td>
+            <td class="ok"><a href="https://httpd.apache.org/docs/2.4/mod/mod_ssl.html#sslciphersuite">yes</a></td>
+            <td class="ok">yes (if using OpenSSL)</td>
```
Comment (igrigorik) [inline index.html]: Any docs for the OpenSSL + 0RTT bit that we can point to?

We don't need the comment, we can just link "yes".

Comment (JBlond) [reply index.html]: There is no link in the apache docs for 0-rtt. I can change it just to yes.

File: index.html [hunk 2]
```diff
@@ -159,7 +159,7 @@ <h3 id="server-performance"><a href="#server-performance">Server performance</a>
             <td class="ok"><a href="https://wiki.mozilla.org/Security/Server_Side_TLS#Apache">yes</a></td>
             <td class="ok"><a href="https://httpd.apache.org/docs/trunk/mod/mod_http2.html">yes</a></td>
             <td class="ok"><a href="https://httpd.apache.org/docs/2.4/mod/mod_ssl.html#sslciphersuite">yes</a></td>
-            <td class="ok">yes (if using OpenSSL)</td>
+            <td class="ok">yes</td>
```
[the remark this answers is not in the document]
Comment (JBlond) [reply index.html]: Well there is https://wiki.openssl.org/index.php/SSL/TLS_Client#0-RTT
But I'm not sure if that is sufficiant

[the remark this answers is not in the document]
Comment (igrigorik) [reply index.html]: Given that there is no guides or docs on how to enable this, I'm inclined to keep it as 'no'.. Until that's fixed. Any objections?

https://wiki.openssl.org/index.php/TLS1.3

[the remark this answers is not in the document]
Comment (JBlond) [reply index.html]: Sure. I changed it to no. WHen I find a solution my self or a valid link I open another PR

[the remark this answers is not in the document]
Comment (igrigorik) [reply index.html]: Great, thank you!

**Pull request merged**